Foodie/foodie.rb

```diff
@@ -0,0 +1,34 @@
+$menu={
+    "PAVBHAJI"=>70,"DOSA"=>30,"CHHOLE BHATURE"=>80,"PULAV"=>60,
+    "PIZZA"=>99,"PASTA"=>65,"MANCHURIAN"=>70,"BURGER"=>50,
+    "UTTAPPA"=>35,"HAKKA NOODLES"=>65,"PANEER CHILLI"=>80
+    }
+
+load 'OrderManager.rb'
+O1=OrderManager.new
+choice="Y"
+while choice=="Y"||choice=="y"
+  puts "----Order Management System----"
+  puts " 1.Take Order"
+  puts " 2.Change Status"
+  puts " 3.Display Order"
+  puts " 4.Exit"
+  print "Which Operation: "
+  op_no = gets.to_i
+  case op_no
+  when 1
+    O1.takeOrder
```

Comment:
ruby method  name should be in `snake_case`  change all method names. also use better variable names than O1. check variable naming convensions of ruby. When to start with small case or upcase